theme_test_custo/static/src/builder/background_shape_option_plugin.js

```diff
@@ -0,0 +1,20 @@
+import { BackgroundShapeOptionPlugin } from "@website/builder/plugins/background_option/background_shape_option_plugin";
+import { patch } from "@web/core/utils/patch";
+import { _t } from "@web/core/l10n/translation";
+
+patch(BackgroundShapeOptionPlugin.prototype, {
+    getBackgroundShapeGroups() {
+        const bgShapeGroups = super.getBackgroundShapeGroups();
+        bgShapeGroups.basic.subgroups["custom_shape"] = {
+            label: "Custom Shapes",
+            shapes: {
+                "theme_test_custo/curves/01": {
+                    selectLabel: _t("Curve 01"),
+                    transform: false,
+                    togglableRatio: true,
+                }
+            },
+        };
+        return bgShapeGroups;
+    },
+});
```

Comment:
How come this commit adds 80+ lines without removing/changing anything to fix a test? :thinking: